### ISSUE TITLE
Unable to retrieve or deploy ManagedTopics

### ISSUE THREAD

**Issue by colin-jarvisSFDC** [forcedotcom/cli#1192]
### Summary

Starting with version 7.117.0, when deploying ManagedTopics metadata using force:source:deploy, I am seeing multiple instances of the following error message.  It seems to be one message for each topic in the source file.

> An object '_topicName_' of type ManagedTopic was named in package.xml, but was not found in zipped directory.

Each error message would have _topicName_ replaced with the full name of a single topic from within the ManagedTopics metadata file.

### Steps To Reproduce:

1. Create an org with managed topics.
1. Retrieve the metadata using `sfdx force:source:retrieve -m ManagedTopics`.   At this step, you may get an UNKNOWN_EXCEPTION error during retrieval.
1. If retrieving was successful, or if you have existing ManagedTopics metadata, attempt to deploy using `sfdx force:source:deploy -p <pathToManagedToipcsFile>`

### Expected result

Retrieval and deployment should succeed.

### Actual result

- UNKOWN_ERROR during retrieval.
- Deployment fails at very end with error messages like above.

### System Information

```
{
        "cliVersion": "sfdx-cli/7.118.1",
        "architecture": "win32-x64",
        "nodeVersion": "node-v16.1.0",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.2.3 (core)",
                "@oclif/plugin-not-found 1.2.4 (core)",
                "@oclif/plugin-plugins 1.10.1 (core)",
                "@oclif/plugin-update 1.4.0-3 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.0 (core)",
                "@oclif/plugin-which 1.0.3 (core)",
                "@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "alias 1.1.10 (core)",
                "apex 0.2.8 (core)",
                "auth 1.7.1 (core)",
                "config 1.2.34 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.1 (core)",
                "generator 1.2.0 (core)",
                "limits 1.2.1 (core)",
                "org 1.7.0 (core)",
                "salesforce-alm 52.3.3 (core)",
                "schema 1.0.8 (core)",
                "sfdx-cli 7.118.1 (core)",
                "sfdx-manifest-plugin 0.0.0 (link) C:\\Users\\JarvCo01\\dev\\sfdx-package-xml-generator-plugin",
                "source 1.0.15 (core)",
                "telemetry 1.2.3 (core)",
                "templates 52.1.0 (core)",
                "trust 1.0.6 (core)",
                "user 1.4.0 (core)"
        ],
        "osVersion": "Windows_NT 10.0.19042"
}
```

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-9910605](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000007rhUGYAY/view)

**Comment by shetzel** [comment]
This was broken with https://github.com/forcedotcom/source-deploy-retrieve/pull/446.  It's difficult to know which child types are necessary for a deployment and which aren't.  Relying on a convention works for most types but not all (very unfortunately).  Sorry for the regression.  The fix is super easy so I'll get it in the next CLI release candidate.

A workaround would be to remove the `ManagedTopic` entries from the manifest after a `force:source:convert`, then deploy via `force:mdapi:deploy` command.

**Comment by colin-jarvisSFDC** [comment]
Thanks for the detailed update.

**Comment by colin-jarvisSFDC** [comment]
This appears to be fixed in version 7.118.1.
